weave-js/src/components/PagePanelComponents/Home/Browse2/callTree.ts

```diff
@@ -54,24 +58,9 @@ export interface CallFilter {
   callIds?: string[];
 }
 
-export interface Call {
-  name: string;
-  inputs: {_keys?: string[]; [key: string]: any};
-  output: undefined | {_keys?: string[]; [key: string]: any};
-  status_code: string; // TODO enum
-  exception?: string;
-  attributes: {[key: string]: any};
-  summary: {latency_s: number; [key: string]: any};
-  span_id: string;
-  trace_id: string;
-  parent_id: string;
-  timestamp: number;
-  start_time_ms: number;
-  end_time_ms: number;
-}
-
+export type Call = RawSpanFromStreamTableEra;
```

Comment:
See the new `RawSpanFromStreamTableEra` for comments on why this is named this way and moved